### PR TITLE
Feature/read me

### PR DESCRIPTION
**Pull request by jdursema**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    Click the "refresh repos" button and your nui repos will populate below.
+
+
+2. Add your platform file to booey by entering the absolute path to your platform file in the "platform directory" input. From here you can run the commands `./install` by clicking on the "clean/install" button or `npm run build && npm run start` by clicking on the "build/start" button. You'll be able to see your commands run in the terminal below the input.
+
+
+3. Now that you have added your nui repos and the platform repo, you can copy your local code over to the platform by clicking "build & copy -> build & start" next to on of your repos OR select multiple repos and click the "Build & Copy selected" button above your list of repos. You'll be able to see the commands running in the terminal on the right of your page or in your terminal locally.
+
+
+## Contributing to Booey
```
Comment (mdaxtman) [inline README.md]: I would suggest we add instructions for the dev server as well. `npm run dev`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    * **Multiple Selected Repos** - If you would only like to add a select few repos and not all the nui components in a folder, you can multiple by separating the paths by a semi colon:
+        `"/Users/MyAccount/MyNuiComponents/NameOfComponent; /Users/MyAccount/MyNuiComponents/NameOfOtherComponent"`
+
+    Click the "refresh repos" button and your nui repos will populate below.
+
+
+2. Add your platform file to booey by entering the absolute path to your platform file in the "platform directory" input. From here you can run the commands `./install` by clicking on the "clean/install" button or `npm run build && npm run start` by clicking on the "build/start" button. You'll be able to see your commands run in the terminal below the input.
+
+
+3. Now that you have added your nui repos and the platform repo, you can copy your local code over to the platform by clicking "build & copy -> build & start" next to on of your repos OR select multiple repos and click the "Build & Copy selected" button above your list of repos. You'll be able to see the commands running in the terminal on the right of your page or in your terminal locally.
```
Comment (mdaxtman) [inline README.md]:
```suggestion
3. Now that you have added your nui repos and the platform repo, you can copy your local code over to the platform by clicking "build & copy -> build & start" next to one of your repos OR select multiple repos and click the "Build & Copy selected" button above your list of repos (not yet functional). You'll be able to see the commands running in the terminal on the right of your page.
```
I stopped outputting the std out in the console. I found a typo too. :-).

File: README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    * **Multiple Selected Repos** - If you would only like to add a select few repos and not all the nui components in a folder, you can multiple by separating the paths by a semi colon:
+        `"/Users/MyAccount/MyNuiComponents/NameOfComponent; /Users/MyAccount/MyNuiComponents/NameOfOtherComponent"`
+
+    Click the "refresh repos" button and your nui repos will populate below.
+
+
+2. Add your platform file to booey by entering the absolute path to your platform file in the "platform directory" input. From here you can run the commands `./install` by clicking on the "clean/install" button or `npm run build && npm run start` by clicking on the "build/start" button. You'll be able to see your commands run in the terminal below the input.
+
+
```
Comment (mdaxtman) [inline README.md]: can we add instructions for how to interact with the input boxes? and how the repos input can take in multiple directories? eg. `/path/to/repo; path/to/other/repo`

Comment (jdursema) [reply README.md]: Isn't that what line 14-24 do? Or should I add more?

Comment (mdaxtman) [reply README.md]: ah, never mind. I missed that. Thanks!

**Pull request merged**